etc/profile-a-l/evolution.profile

```diff
@@ -40,7 +72,25 @@ novideo
 protocol unix,inet,inet6
 seccomp
 shell none
+tracelog
 
+disable-mnt
```

Comment:
Is it common to attach files from USB-Sticks?

Reply:
Is it? Idk...